### PR TITLE
feat: added Product page

### PR DESCRIPTION
**Pull request by Tanjum786**
**Added product page **
-added filter section
-added products section
Screenshot from 2022-02-18 00-26-10
Screenshot from 2022-02-18 00-27-27

### CODE REVIEW

File: CSS/product_page.css [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+.landscap_cards,
+.price_info,
+.landscap_btns,
+.price_range,
+.product_card_details,
+.price_list,
+.fillter_container,
+.product_container,
+.product_wishlist_icon,
+.containt_list {
```
Comment (Sameony) [inline CSS/product_page.css]: Create a utility class for this property and use that everywhere

File: CSS/product_page.css [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+.price_list,
+.product_title {
+    justify-content: space-between;
+}
+
+.prices,
+.fillter_container,
+.landscap_cut_price,
+.landscap_cart_off,
+.clear_btn {
```
Comment (krishanu2911) [inline CSS/product_page.css]: try to use a common class for a single property

**Review by Sameony — COMMENTED** [review]
Import and use utility classes from your component library

**Review by krishanu2911 — APPROVED** [review]
Well written code and nice design just added few suggestion

File: CSS/product_page.css (hunk 1 above)
Comment (Tanjum786) [reply CSS/product_page.css]: okay thank you , i will use

File: CSS/product_page.css (hunk 2 above)
Comment (Tanjum786) [reply CSS/product_page.css]: okay .

**Review by Sameony — APPROVED** [review]

**Pull request merged**